### DISCUSSION TITLE
Failed to parse "never"

### DISCUSSION THREAD

**Discussion by mbrevda** [renovatebot/renovate#18265] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

When setting the schedule to "never", the logs show `Failed to parse "never"`. Use case: shared config, want to temporarily stop renovate from opening new PRs.

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: prAlreadyExisted=false(branch="renovate/octokit-request-action-2.x")
DEBUG: Checking schedule(never, xxx/xxx)(branch="renovate/octokit-request-action-2.x")
WARN: Invalid schedule: Failed to parse "never"(branch="renovate/octokit-request-action-2.x")
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

### Later test

When testing this in Later, it seems to parse just fine. Can be run [here](https://npm.runkit.com/%40breejs%2Flater)

```js
var later = require("@breejs/later")
var s = later.parse.text('never');

later.schedule(s).next(1);
```

**Comment by mbrevda** [comment]
Closing for now, will try `on never`

**Comment by rarkins** [comment, marked as answer]
Schedule field isn't intended for disabling updates. Instead set enabled=false

**Reply by mbrevda** [reply]
During certain periods, we want to disable all updated except for our internal packages. Can `enabled` be overwritten by `packageRules`?

**Reply by viceice** [reply]
yes, you can do that via package rule

**Reply by mbrevda** [reply]
Not seeing that in the [docs](https://docs.renovatebot.com/configuration-options/#packagerules)?

**Reply by rarkins** [reply]
`enabled` is a field available for use in and outside `packageRules`

**Reply by mbrevda** [reply]
Thanks, will give it a try!

**Reply by HonkingGoose** [reply]
> Not seeing that in the [docs](https://docs.renovatebot.com/configuration-options/#packagerules)?

You're looking in the wrong place. 😉 Check out the `enabled` docs: [^docs]

> To disable Renovate for all `eslint` packages, you can configure a package rule like:
> 
> ```json
> {
>   "packageRules": [
>     {
>       "matchPackagePatterns": ["^eslint"],
>       "enabled": false
>     }
>   ]
> }
> ```

---

Edit: I created a PR to direct our users to the correct feature:

- #18269

[^docs]: https://docs.renovatebot.com/configuration-options/#enabled

**Reply by mbrevda** [reply]
Just tried this: I set `enabled: false` in the general section, then tried overriding in `packageRules`, but now renovate wont run at all. The logs show `INFO: Repository is disabled - skipping`.

To clarify: I'm looking to programmatically disable renovate in general for given days, but keep it enabled for specific packages

**Reply by viceice** [reply]
use a general package rule for all with some excluded packages and use enabled only there
